test/e2e/scalability/density.go

```diff
@@ -226,8 +226,13 @@ func density30AddonResourceVerifier(numNodes int) map[string]framework.ResourceC
 
 func logPodStartupStatus(c clientset.Interface, expectedPods int, observedLabels map[string]string, period time.Duration, stopCh chan struct{}) {
 	label := labels.SelectorFromSet(labels.Set(observedLabels))
-	podStore := testutils.NewPodStore(c, metav1.NamespaceAll, label, fields.Everything())
+	podStore, err := testutils.NewPodStore(c, metav1.NamespaceAll, label, fields.Everything())
+	if err != nil {
+		framework.Logf("Couldn't initialize pod store: %v", err)
```

Comment:
Don't we want to framework.ExpectNoError() here?

Reply:
done